src/main/java/rs/hybridit/model/User.java

```diff
@@ -57,4 +75,41 @@ public User(UserDto userDto) {
 
 	}
 
+	public void setAuthority(Role role){
+		List<Authority> authorities = new ArrayList<>();
+		Authority a = new Authority();
+		a.setName(role);
+		authorities.add(a);
+		this.setAuthorities(authorities);
+	}
+
+	@Override
+	public Collection<? extends GrantedAuthority> getAuthorities() {
+		return null;
```

Comment:
why null?

Reply:
Mistake.

Reply:
As I changed User model so that it now has field Authority instead of list<Authority> i will leave this method like this (I still need to have it as I imeplement UserDetails inteface).